pkg/redfish/rfendpoints.go

```diff
@@ -975,6 +975,11 @@ func (ep *RedfishEP) GetRootInfo() {
 	// Next, the PowerEquipment for the endpoint, if it exits.  For now,
 	// we just get the RackPDUs collection under it.
 	//
+	// HPE PDUs use PowerDistribution, so setup PowerEquipment path
+	if ep.ServiceRootRF.PowerDistribution.Oid != "" {
+		ep.ServiceRootRF.PowerEquipment.Oid = "/redfish/v1/PowerEquipment"
```

Comment:
This probably should be the following just in case the URL stops being `/redfish/v1/PowerEquipment` for whatever reason and because redfish is supposed to be self-describing. The `TrimSuffix()` will take care of the trailing `/`.: 
```
ep.ServiceRootRF.PowerEquipment.Oid = strings.TrimSuffix(ep.ServiceRootRF.PowerDistribution.Oid, "/")
```

Reply:
Hmmm... looking at our internal PDUs, it seems like it is more than just removing the trailing `/`. These PDUs actually tack on the next item `/RackPDUs` because it is the only one (which might change). Maybe just `TrimSuffix(s, "/RackPDUs")` for now?

Reply:
I would like to keep this as making the change would require retesting the functionality - if (when) the HPE PDU change their redfish responses, we will most likely need to change several items.